src/test/java/edu/harvard/iq/dataverse/api/UtilIT.java

```diff
@@ -1881,6 +1881,9 @@ static Response getDatasetVersions(String idOrPersistentId, String apiToken, Int
             requestSpecification = given()
                     .header(UtilIT.API_TOKEN_HTTP_HEADER, apiToken);
         }
+        String io = "/api/datasets/" + idInPath + "/versions" + optionalQueryParam;
+        System.out.println(io);
```

Comment:
Do we need to keep these debugging lines?

Reply:
I need to remove those, thanks!

Reply:
I have removed those, thanks!